src/fal/cli/flow_runner.py

```diff
@@ -83,47 +84,41 @@ def node_to_script(node: Union[FalFlowNode, None], fal_dbt: FalDbt) -> FalScript
         raise Exception(f"Cannot convert node to script. Node: {node}")
 
 
-def _combine_fal_run_results(target_path):
-    result_files = _get_fal_run_results(target_path)
+def _combine_fal_run_results(target_path: str) -> None:
+    target_path = Path(target_path)
+    dbt_run_results, fal_run_results = [], []
+    for path in target_path.glob("fal_results_*.json"):
+        assert path.is_file()
 
-    if not result_files:
-        return
+        results = _get_all_result_content(path)
 
-    all_results = reduce(
-        lambda all, next: all.extend(_get_result_array(next)) or all,
-        result_files,
-        [],
-    )
+        if results.get(SCHEMA_KEY) != "fal":
```

Comment:
Nit: Maybe we put "fal" in a variable?

Reply:
this is checking the `"schema"` property in the `fal_results_X.json` file. If it DOES NOT have `"schema": "fal", we process it as a `dbt` run_results artifact.

Reply:
which gets me thinking, @isidentical , this is built for `"dbt_schema_version": "https://schemas.getdbt.com/dbt/run-results/v4.json"`. Maybe we should check that it is that scema version?

Reply:
Makes sense!